examples/midnight-basic-example/midnight-api-examples.md

```diff
@@ -0,0 +1,161 @@
+# Midnight Relayer CURL Examples
```

Comment:
We usually do full Docker example + README file in tutorial mode.

Maybe we should do it here as well because this could be main entry point for users that would like to spin up service and try it put. They just follow tutorial and then run docker commands to spin it up.